sdks/python/apache_beam/testing/load_tests/combine_test.py

```diff
@@ -138,6 +143,11 @@ def setUp(self):
     else:
       self.fanout = int(self.fanout)
 
+    self.top_count = self.pipeline.get_option('top_count')
+    if self.top_count is None:
+      self.fail('You should set \"--topCount\" option to use TOP combiners')
+    self.top_count = int(self.top_count)
```

Comment:
What do you think about changing this and above construction to just try int() and then catch Type and Value exception? (for cases it's None or not a number value)